src/app/_components/post-card/post-card.ts

```diff
@@ -46,7 +45,12 @@ export class PostCardComponent {
   getComments() {
     this.apiService.getPostComments(this.post.id).subscribe(res => {
       this.comments = res;
-    })
+      for (let comment of this.comments) {
+        this.apiService.getAccount(comment.AuthorId).subscribe(response => {
+          comment['displayName'] = response.body.displayName;
+        });
+      }
```

Comment:
I modified this section because we actually need to modify comments 'in-place'. The way it was before re-added comments to an array after their author info was looked up. The problem with that is that there's no guarantee that those requests will resolve in the same order we made them in, which means that the comments would have been in author-loaded-order, not the order originally returned by the server.